examples/website-demos/social-messaging/src/components/ChatContainer/ChatContainer.tsx

```diff
@@ -58,7 +60,7 @@ export type UserType = { image?: string };
 export const ChatContainer: React.FC = () => {
     const [chatClient, setChatClient] = useState<StreamChat | null>(null);
```

Comment:
i think you can just do this...
```suggestion
    const [chatClient, setChatClient] = useState<StreamChat>();
```